packages/ui-app/src/InputTags.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017-2019 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps } from './types';
+
+import React from 'react';
+import store from 'store';
+
+import Dropdown from './Dropdown';
+
+type Option = {
+  key: string,
+  text: string,
+  value: string
+};
+
+type Props = BareProps & {
+  allowAdd?: boolean,
+  defaultValue?: Array<string>,
+  help?: React.ReactNode,
+  isDisabled?: boolean,
+  isError?: boolean,
+  label?: React.ReactNode,
+  onChange?: (value: Array<string>) => void,
+  placeholder?: string,
+  value?: Array<string>
+};
+
+type State = {
+  options: Array<Option>
+};
+
+function loadTags (): Array<string> {
+  return store.get('tags') || ['Default'];
```

Comment:
Any reason to have a `Default` tag?

Reply:
I just wanted "something" so you don't start off with everything completely blank. (The way the SUI add-your-won dropdown works it is def. not intuitive)